test/bigtest/tests/units/AllJobLogs-test.js

```diff
@@ -0,0 +1,187 @@
+import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { noop } from 'lodash';
+import { expect } from 'chai';
+import Pretender from 'pretender';
+import { cleanup } from '@bigtest/react';
+import {
+  describe,
+  before,
+  beforeEach,
+  afterEach,
+  it,
+} from '@bigtest/mocha';
+
+import {
+  mountWithContext,
+  buildResources,
+  buildMutator,
+} from '@folio/stripes-data-transfer-components/interactors';
+import CalloutInteractor from '@folio/stripes-components/lib/Callout/tests/interactor';
+import { StripesContext } from '@folio/stripes-core/src/StripesContext';
+import { Paneset } from '@folio/stripes/components';
+import commonTranslations from '@folio/stripes-data-transfer-components/translations/stripes-data-transfer-components/en';
+
+import translations from '../../../../translations/ui-data-export/en';
+import { allLogsPaneInteractor } from '../../interactors';
+import { AllJobLogsViewComponent } from '../../../../src/components/AllJobLogsView';
+import {
+  OverlayContainer,
+  translationsProperties,
+} from '../../helpers';
+import { logJobExecutions } from '../../network/scenarios/fetch-job-executions-success';
+
+const {
+  searchResults,
+  header,
+} = allLogsPaneInteractor.searchAndSort;
+const { getCellContent } = searchResults;
+const getUser = row => logJobExecutions[row].runBy;
+
+describe('All job logs view', () => {
+  before(async () => {
+    await cleanup();
+  });
+
+  beforeEach(async () => {
+    window.history.pushState({}, document.title, '/?sort=-completedDate');
+  });
+
+  afterEach(async () => {
+    window.history.pushState({}, document.title, '/?sort=-completedDate');
+  });
+
+  beforeEach(async () => {
+    await mountWithContext(
+      <StripesContext.Provider
+        value={{
+          okapi: { url: '' },
+          logger: { log: noop },
+        }}
+      >
+        <Router>
+          <Paneset>
+            <AllJobLogsViewComponent
+              resources={buildResources({
+                resourceName: 'jobExecutions',
+                records: logJobExecutions,
+              })}
+              mutator={buildMutator()}
+            />
+          </Paneset>
+          <OverlayContainer isFullScreen={false} />
```

Comment:
Please, could you clarify why the OverlayContainer shouldn't use full screen settings in this case? As far as I know, it usually use these settings on envs.

Reply:
I found that in full-screen mode for in this case the `OverlayContainer` overlaps the whole page so there is no way to click buttons. For other cases, there is no such problem.

Reply:
The property pointer-events: none was added to overlay container. Can it fix our problem?

Reply:
Yes, it does. Thanks. Removed property.